pebbles/server.py

```diff
@@ -119,7 +119,19 @@
 
     @sso.login_handler
     def login(user_info):
-        eppn = user_info['eppn']
+        if user_info['authmethod'] == app.config['CSC_LOGIN']:
+            if user_info['accountlock'] and user_info['accountlock'] == 'true':
+                error_description = 'Your CSC account have been locked. \
+                    You were not authorized to access. Contact the administrator'
+                return render_template(
+                    'error.html',
+                    error_title='User not authorized',
+                    error_description=error_description
+                )
+            else:
+                eppn = user_info['eppn'].split('@')[0] + '@cscuserid'
```

Comment:
I think this deserves an explanation, either in the commit message, as a comment or maybe both